Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -578,13 +578,13 @@ Don't have a 4chan Pass?<br>
     <string name="tap_image_for_extra_options">Tap image for extra options</string>
 
     <string name="settings_import_export">Import/Export</string>
-    <string name="settings_import_export_description">Import/export settings/pins, etc.</string>
+    <string name="settings_import_export_description">Import/export settings/PINs, etc.</string>
     <string name="import_or_export_settings">Import/export settings</string>
     <string name="export_settings">Export settings</string>
     <string name="export_settings_to_a_file">Export settings to a file</string>
     <string name="import_settings">Import settings</string>
     <string name="import_settings_from_a_file">Import settings from a file</string>
-    <string name="successfully_exported_text">Exported successfully"</string>
+    <string name="successfully_exported_text">Exported"</string>
```

Comment:
Why is there a quotation mark at the end of this?

Reply:
There isn't…?